README.md

```diff
@@ -439,6 +439,18 @@ pa11y('http://example.com/', {
 
 Defaults to an empty array.
 
+### `ignoreUrl` (boolean)
+
+Whether to use the provided [Puppeteer Page instance][puppeteer-page] as is or use the provided url.
+
+```js
+pa11y('http://example.com/', {
+    ignoreUrl: true
```

Comment:
Can you add `page` to show it's required to pass these two in when using `ignoreUrl`.
Otherwise this looks great 🎉 . Should be good to merge after that tiny change!

Reply:
@joeyciechanowicz def! I'll try and get to it this week. Thanks!

Reply:
@joeyciechanowicz added both `page` and `browser` to the config and merged in master. Lmk if there's anything else thanks!.